mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/contact.jsp

```diff
@@ -0,0 +1,78 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<div class="list-container">
+
+	<%@ include file="/header.jspf" %>
+
+	<div class="form-container">
+		<h3>
+			<liferay-ui:message key="contact-us" />
+
+			<i class="icon-envelope"></i>
+		</h3>
+
+		<portlet:actionURL name="submitContactForm" var="formActionURL">
+			<portlet:param name="redirect" value="/contact.jsp"></portlet:param>
+		</portlet:actionURL>
+
+		<liferay-ui:success key="success" message="success" />
+
+		<liferay-ui:error key="error" message="error" />
+
+		<aui:form action="<%= formActionURL %>" cssClass="contact-form" method="post" name="contact">
```

Comment:
Move <aui:form> to the root of the html, as requested in https://github.com/markocikos/liferay-internship/pull/164#discussion_r128983103.

The idea of the convention is:  if a form is on the page, it should be the main, the base element of the page. Any wrapper should be either in parent jsp or in theme.